lib/mayaUsd/fileio/jobs/writeJob.cpp

```diff
@@ -556,7 +556,10 @@ bool UsdMaya_WriteJob::_FinishWriting()
         UsdGeomSetStageMetersPerUnit(mJobCtx.mStage, mJobCtx.mArgs.metersPerUnit);
     }
 
-    if (usdRootPrim) {
+    if (!mJobCtx.mArgs.defaultPrim.empty()) {
+        defaultPrim = TfToken(mJobCtx.mArgs.defaultPrim);
+        mJobCtx.mStage->GetRootLayer()->SetDefaultPrim(defaultPrim);
```

Comment:
This will handle the case of user selecting "None" in export, because writing `defaultPrim = "None"` means no defaultPrim